dashboard/src/components/ui/table.tsx

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react';
 
 import { cn } from '../../lib/utils';
+import HandleLink from '../HandleLink/HandleLink';
```

Comment:
I know we have some imports that are like `../` but please don't use that, import from `@/components`  
we already have a ticket for adding this eslint rule, the reason we had that is because our vite config was not supporting that in the beginning